packages/runtime-core/src/renderer.ts

```diff
@@ -1201,6 +1201,7 @@ function baseCreateRenderer(
         parentComponent,
         parentSuspense
       ))
+    instance.container = container
```

Comment:
I'm not sure if this usage scenario is common, but I don't think all component instances need to set it
```suggestion
    instance.isCE && (instance.container = container)
```

Reply:
Sure, the scope can be expanded any time in the future if needed.